hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/lock/OmReadOnlyLock.java

```diff
@@ -42,7 +41,7 @@ public OMLockDetails acquireReadLocks(Resource resource, Collection<String[]> re
 
   @Override
   public OMLockDetails acquireWriteLock(Resource resource,
-      String... resources) {
+                                        String... resources) {
```

Comment:
Please change your IDE settings to avoid changing indent in method signatures like that.

Reply:
done thanks for the tip!

Reply:
Uncheck `Method declaration parameters - Align when multiline` like this:

![image](https://github.com/user-attachments/assets/b9ae2b4f-0154-4f9e-95b7-577c640ad9bf)

Reply:
@smengcl Thanks for the tip, I was using the Ozone `checkstyle.xml`, and it seems this is checked by default. Maybe we can fix it in `checkstyle.xml` instead.

Reply:
yeah we should fix checkstyle.xml

Reply:
https://github.com/apache/ozone/pull/8494 please take a look at this pr